kibanaDockerfile

```diff
@@ -0,0 +1,2 @@
+FROM docker.elastic.co/kibana/kibana:6.4.0
```

Comment:
Just for future use/consistency, put this into the templates directory anyway in case we to end up hooking up a variable to something.